client/src/components/PillIcon/PillIcon.css

```diff
@@ -0,0 +1,28 @@
+.pill-container {
+  height: 3.434vw;
+  width: 11.566vw;
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  border-radius: 0.152vw;
+  box-shadow: 0px 0.16vw 0.16vw rgba(0, 0, 0, 0.25);
+}
+img {
+  position: absolute;
+  align-self: auto;
+  width: 1.869vw;
+  height: 1.465vw;
+  left: 9.639vw;
+}
+h1 {
+  font-family: sans-serif;
+  color: white;
```

Comment:
Can you change this to use the CSS variables for the colors? For example
color: var(--white);

Reply:
thank you. i changed the variable name for the h1 tag to a variable